aplt/runner.py

```diff
@@ -283,7 +283,10 @@ def run_scenario(args=None, run=True):
     scenario = locate_function(arg)
     log.startLogging(sys.stdout)
     statsd_client = parse_statsd_args(arguments)
-    scenario_args = try_int_list_coerce(arguments["SCENARIO_ARGS"])
+    scenario_args = arguments["SCENARIO_ARGS"]
+    if scenario_args:
+        parts = [x.strip() for x in scenario_args.strip().split(",")]
+        scenario_args = try_int_list_coerce(parts)
```

Comment:
A bit confused here, why isn't this in try_int_list_coerce? You repeat the pattern for both instances of the call.